Kickstarter-iOS/Views/Controllers/CommentsViewController.swift

```diff
@@ -212,6 +214,18 @@ extension CommentsViewController: CommentTableViewFooterViewDelegate {
   }
 }
 
+// MARK: - UITableViewDelegate
+
+extension CommentsViewController {
+  override func tableView(_: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+    if self.dataSource.isErrorStateIndexPath(indexPath) {
+      return (self.view.safeAreaLayoutGuide.layoutFrame.height * 0.70)
+    }
+
+    return UITableView.automaticDimension
```

Comment:
I may be out of the loop here but where is 0.70 coming from?

Reply:
`0.70` is just 70% of the parent view's height. Full height was showing the scrolling indicator when error state cell is shown and I wanted to prevent that.

Reply:
Ah ok is that a product requirement? I would leave a comment in place, if so, otherwise it seems like a magic number.